web/src/i18n.js

```diff
@@ -0,0 +1,46 @@
+import i18n from 'i18next';
```

Comment:
Can you make this a TypeScript file? End it with the `*.ts` instead of the JavaScript extension.